extensions/visualizations/visualizations.js

```diff
@@ -73,8 +73,7 @@ oppia.directive('oppiaVisualizationFrequencyTable', [
         function($scope, $attrs, HtmlEscaperService) {
           $scope.data = HtmlEscaperService.escapedJsonToObj($attrs.data);
           $scope.options = HtmlEscaperService.escapedJsonToObj($attrs.options);
-          $scope.isAddressed = HtmlEscaperService.escapedJsonToObj(
-            $attrs.isAddressed);
+          $scope.showAddressedInfo = $attrs.showAddressedInfo;
```

Comment:
Let's go with $scope.addressedColumnIsShown -- every time I read "showAddressedInfo" it sounds more like a function than a boolean.